### PR TITLE
added kawaiCat theme with decorations

### PR DESCRIPTION
**Pull request by ayazhankadessova**
Calming blend of soft pinks and teals with cat-themed decorations (gentle paw prints, twinkling stars, and delicate borders) !

Screenshot 2024-12-07 at 11 50 34 AM

### CODE REVIEW

**Comment by bobbibu13** [comment]
So cute!

File: src/decorations.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+interface KawaiiDecorationsProps {
+  color: string
+}
+
+export function KawaiiDecorations({ color }: KawaiiDecorationsProps) {
```
Comment (JacobLinCool) [inline src/decorations.tsx:5]: I think the decoration layer is a good idea! I also believe that we’ll see more types of decorations from the community. Therefore, I would prefer to use the same name as this theme (`KawaiiCatDecoration`), so we can add more cute decorations in the future without naming conflict.

**Comment by JacobLinCool** [comment]
You can ignore the CI failure; it's only relevant to the bad authentication on the token. (It seems that we can only have read permissions when running from forks.) We may need to find a new place to temporarily host the image...

ref: https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

File: src/decorations.tsx (hunk 1 above)
Comment (ayazhankadessova) [reply src/decorations.tsx]: makes sense, i fixed the naming!

**Comment by JacobLinCool** [comment]
card

**Review by JacobLinCool — APPROVED** [review]
LGTM

**Pull request merged**